packages/office-ui-fabric-react/src/components/Button/BaseButton.tsx

```diff
@@ -37,6 +40,14 @@ interface IButtonClassNames {
 
 export class BaseButton extends BaseComponent<IBaseButtonProps, IBaseButtonState> implements IButton {
 
+  private get _splitButton(): boolean {
+    return (this.props.menuProps != null && this.props.onClick != null);
```

Comment:
Oh I like that! I actually had it this way before, but removed it because I thought that it wasn't common practice in TS to use `is...`.